_posts/refarch2-guides/eks.adoc

```diff
@@ -0,0 +1,31 @@
+---
+title: [title]
+categories: Upgrades
+image: /assets/img/guides/refresh_icon.png
+excerpt: Learn how to update your v1 Reference Architecture to use the Gruntwork Service Catalog.
+tags: ["aws", "terraform", "terragrunt"]
+cloud: ["aws"]
+redirect_from: /static/guides/upgrades/how-to-update-your-ref-arch/
+---
+:page-type: guide
+:page-layout: post
+
+:toc:
+:toc-placement!:
+
+// GitHub specific settings. See https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74 for details.
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+toc::[]
+endif::[]
+
+== EKS Migration Guides
+
+The migration process for the EKS cluster, core services, and applications from Reference Architecture v1 (uses
+infrastructure-modules) to v2 (uses https://github.com/gruntwork-io/terraform-aws-service-catalog[Service Catalog]) is
+slightly different from migrating other modules because the ordering matters and because some post-migration steps are
+required.
```

Comment:
This is WIP because I wanted to just give a taste of it rather than put a lot of time into it.

Basically, it would describe how to migrate all 4 of the EKS modules in order, but also highlight the places where you need to run apply differently, or run some other things post-apply. This is the case for EKS cluster and EKS core services.

It's also the case that they will have to hardcode the OIDC arn/url the first go-around, and once everything's migrated, they can go back and use the dependency output of EKS cluster (the last thing they'll migrate) to replace those hardcodes. 

I can imagine it being confusing to go back and forth between the main article and these special-cases, so I would like a separate "main article" listing the high-level steps for just the EKS stuff.

Reply:
Yea that seems reasonable!

Reply:
@yorinasub17 Input on this? Will do in https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/pull/50

Reply:
Done, resolving!